flutter_twilio_conversations/pubspec.yaml

```diff
@@ -1,18 +1,22 @@
 name: flutter_twilio_conversations
 description: Integrate the Twilio Conversations SDK with your Flutter app using this Twilio Conversations Flutter plugin
-version: 1.1.7+9
+version: 2.0.0+10
 repository: https://github.com/Diversido/flutter_twilio_conversations
 issue_tracker: https://github.com/Diversido/flutter_twilio_conversations/issues
 
 environment:
   sdk: ">=2.14.0 <4.0.0"
-  flutter: '>=1.17.0'
+  flutter: ">=1.17.0"
 
 dependencies:
   flutter:
     sdk: flutter
-    
+  flutter_twilio_conversations_platform_interface:
+    path: ../flutter_twilio_conversations_platform_interface
```

Comment:
Hey @asmodeoux,  do you how can this can be published since `path`  is not allowed for published plugins?

Reply:
I think we have to split it and publish the interface package separately